_pages/platform/pipeline.md

```diff
@@ -12,29 +12,32 @@ permalink: /platform/pipeline/
 
-Algorithmia was built from the ground up with machine learning at scale use cases in mind. Pipelining is a key part of any full scale deployment solution. Teams need to be able to productionize models as parts of a whole. This means two things: first, splitting up your machine learning workflows into independent, reusable, modular parts that can then be pipelined together to create models. This makes building models more efficient, cutting out redundant work. Second, we want to automate the machine learning workflow by enabling data to be transformed and correlated into a model that can then be analyzed to achieve outputs, streamlining the process of inputting data into the ML model.
+Algorithmia was built from the ground up to support machine learning at scale, and pipelining is a key part of any full-scale deployment solution. Teams must be able to productionize individual models as parts of a larger workflow, which means two things:
 
-A pipelining architecture solves the problems that arise at scale:
+1. Complex machine learning workflows should be divided into independent, reusable, modular parts that can then be connected to form a complete pipeline. This sort of modularity makes data scientists and engineers more efficient, cutting out redundant work as the component pieces are reusable and easy to work with.
+2. Complex machine learning workflows should be automated. This can be achieved by streamlining the process of transforming data, inputting it into a model where inference happens, and shuttling the output to the next step in the pipeline.
+
+When machine learning is run at scale, issues arise related to the large volume of data processed, the variety of services used, and the versions of these services. You can employ the following pipelining strategies to address these issues:
 
 - Volume: only call parts of the workflow when you need them, and cache or store results that you plan on reusing.
-- Variety: when you expand your model portfolio, you can use pieces of the beginning stages of the workflow by simply pipelining them into the new models without replicating them.
-- Versioning: when services are stored in a central location and pipelined together into various models, there is only one copy of each piece to update. All instances of that code will update when you update the original.
+- Variety: when you expand your model portfolio, you can use components from the beginning stages of existing workflows by simply pipelining them with your new models instead of actually replicating the code.
+- Versioning: when services are stored in a central location and pipelined together with various models, there's only one copy of each piece of the pipeline to update. Because services in multiple different pipelines may be calling the same upstream service, when the original upstream services is updated, all downstream services will have access to that updated version immediately, and code doesn't need to be changed in multiple places.
```

Comment:
```suggestion
- Versioning: when services are stored in a central location and pipelined together with various models, there's only one copy of each piece of the pipeline to update. Because services in multiple different pipelines may be calling the same upstream service, when the original upstream service is updated, all downstream services will have access to that updated version immediately, and code doesn't need to be changed in multiple places.
```